src/elements/content-sidebar/Sidebar.js

```diff
@@ -155,7 +177,7 @@ class Sidebar extends React.Component<Props, State> {
                             isOpen={isOpen}
                             key={file.id}
                             metadataSidebarProps={metadataSidebarProps}
-                            onVersionHistoryClick={onVersionHistoryClick}
+                            onVersionHistoryClick={onVersionHistoryClick || handleVersionHistoryClick}
```

Comment:
Is this logic to defer to the user of the ContentSidebar element in case they have provided `onVersionHistoryClick` otherwise to fall back to this new feature if enabled?

Reply:
Yep!